.github/workflows/release-go-task.yml

```diff
@@ -88,11 +88,11 @@ jobs:
         run: |
           wget -q -P /tmp https://github.com/fsaintjacques/semver-tool/archive/3.2.0.zip
           unzip -p /tmp/3.2.0.zip semver-tool-3.2.0/src/semver >/tmp/semver && chmod +x /tmp/semver
-          if [[
+          if [[ \
             "$(
               /tmp/semver get prerel \
                 "${GITHUB_REF/refs\/tags\//}"
-            )"
+            )" \
```

Comment:
This fix is 100% correct, but as I was evaluating it, it struck me that this test code is very difficult to understand. Maybe we can take this opportunity to make the code more clear while fixing the bug.

Even though I know I've looked at this multiple times before, I still had the intuition it was testing the exit status of the `semver` command. I had to actually check to determine that the behavior of the `semver` tool is to print a version string if the argument is a pre-release version, and nothing if it is not. So the test is whether a string is returned.

So I have an alternative proposal:

```sh
          if [[
            "$(
              /tmp/semver get prerel \
                "${GITHUB_REF/refs\/tags\//}"
            )" != \
            ""
          ]]; then
            echo "IS_PRE=true" >> $GITHUB_OUTPUT
          fi
```

I think this explicit comparison makes it much easier to understand the code. What do you think Matteo?

Reply:
Looks great, thanks for the input Per! I will make the suggested changes.